### PR TITLE
Add distribution v2211.0 description

### PR DESCRIPTION
**Pull request by MakisH**
Currently waiting for:
- [x] the release of the tutorials to be formally published
- [x] https://github.com/precice/vm/pull/50 to be approved and merged, and a release to be formally tagged and the artifact to be published on Vagrant cloud
- [x] this website to be tagged

### CODE REVIEW

**Comment by MakisH** [comment]
Everything has now been released, and this is only the formal step of freezing the versions set.

File: pages/docs/installation/installation-distribution.md [hunk 1]
```diff
@@ -14,13 +14,39 @@ publishing here lists of known-to-work versions.
 
 Releases of the preCICE distribution are irregular. The version of each distribution is `yymm.r`, reflecting the year, the month, and the revision (bugfixes) of the distribution. Bindings versions reflect compatibility with a specific preCICE version, while adapters use a completely independent versioning scheme. The tutorials follow a `yyyymm.r` scheme and are targeting the released versions of each component. The VM version is based on the tutorials version, followed by the VM revision. While the distribution uses two year digits for convenience, the tutorials and the VM use four digits to allow version comparisons with previous releases that already used four digits.
 
+## v2211.0
+
+preCICE distribution v2211.0 (publishing to DaRUS pending)
+
+This is the last release that works with preCICE v2.x and in the context of the [preCICE Workshop 2023](precice-workshop-2022.html).
```
Comment (uekerman) [inline pages/docs/installation/installation-distribution.md]:
```suggestion
This is the last release that works with preCICE v2.x.
```
I am not sure what "in the context of ..." should actually mean. "The last" is then definitely odd, no?

File: pages/docs/installation/installation-distribution.md [hunk 2]
```diff
@@ -14,13 +14,39 @@ publishing here lists of known-to-work versions.
...
+  - CalculiX adapter: [v2.20.0](https://github.com/precice/calculix-adapter/releases/tag/v2.20.0)
+  - code_aster adapter: commit [ce995e0](https://github.com/precice/code_aster-adapter/tree/ce995e0c41b26fe891ce04fd47fd52cbeff854e9) (same as in v2202.0)
+  - deal.II adapter: [dbb25be](https://github.com/precice/dealii-adapter/commit/dbb25bea51531b7e4e0c9b5e4def3a7fadf8367c)
+  - DUNE adapter: commit [5f2364d](https://github.com/precice/dune-adapter/tree/5f2364d57b517698914cb1d5f9979efe692d9254) (same as in v2202.0, experimental)
+  - FEniCS adapter: [v1.4.0](https://github.com/precice/fenics-adapter/releases/tag/v1.4.0)
+  - OpenFOAM adapter: [v1.2.0](https://github.com/precice/openfoam-adapter/releases/tag/v1.2.0)
+  - SU2 adapter: [ab84387](https://github.com/precice/su2-adapter/tree/ab843878c1d43302a4f0c66e25dcb364b7787478) (same as in v2202.0)
+- Tutorials: [v202211.0](https://github.com/precice/tutorials/releases/tag/v202211.0)
+- vm: [v202211.0.0](https://github.com/precice/vm/releases/tag/v202211.0.0)
+- Website and documentation: [v202211.0.0](https://github.com/precice/precice.github.io/releases/tag/v202211.0.0)
```
Comment (uekerman) [inline pages/docs/installation/installation-distribution.md]:
```suggestion
- Website and documentation: [v202211.0.0](https://github.com/precice/precice.github.io/releases/tag/v202211.0.0), [libprecice2_2.2.5_docs_v202211.0.0.pdf](https://github.com/precice/precice.github.io/releases/download/v202211.0.0/libprecice2_2.2.5_docs_v202211.0.0.pdf)
```
Could be a nice fit here? We still don't have it anywhere else, right? I also added it to DaRUS last time already.

**Review by uekerman — APPROVED** [review]
Looks good. I checked most of the versions (correct version, correct link).
DaRUS will come after this PR, right? Otherwise, this will be blocked for too long.

File: pages/docs/installation/installation-distribution.md (hunk 1 above)
Comment (MakisH) [reply pages/docs/installation/installation-distribution.md]: I meant that this is what we will use for the workshop (which I wanted to advertise also there). But I see that it is odd.

File: pages/docs/installation/installation-distribution.md (hunk 2 above)
Comment (MakisH) [reply pages/docs/installation/installation-distribution.md]: Indeed, good point! Previously we were checking this in the repository, but not making it discoverable anywhere (apart from DaRUS).

**Comment by MakisH** [comment]
> DaRUS will come after this PR, right? Otherwise, this will be blocked for too long.

So far, we always had DaRUS coming later and updating the badge.

**Pull request merged**